microsoft-edge/devtools-protocol/0.1/http.md

```diff
@@ -0,0 +1,69 @@
+# HTTP Endpoints
+
+**Microsoft Edge DevTools Protocol Version 0.1** supports the following HTTP endpoints. See [using the protocol](../index.md#using-the-protocol) for more on connecting to a browser content process and the [Domains](index.md) documentation for the full web sockets-based devtools protocol API.
```

Comment:
I think we should remove the link from the Domains, at the moment the link would just move the user to a related page, but still wouldn't highlight the domains.

Reply:
Disregard. We will be adding a domains summary page which we will link to.